charts/console/templates/ingress.yaml

```
@@ -38,7 +38,9 @@ spec:
   {{- end }}
   rules:
     {{- range .Values.ingress.hosts }}
-    - host: {{ tpl (.host) $ | quote }}
+    - {{- if .host }}
+      host: {{ .host }}
+      {{- end}}
```

This was previously allowing .host to be a template. This changes removes this feature.
```suggestion
    {{- $host := tpl .host $ }}
    - {{- if $host }}
      host: {{ $host }}
      {{- end}}
```

---

thanks for the review, I've re-added the `tpl` , I can add the `$host` also if you want, but for now I just put the previous code back

---

I was thinking that if `tpl .host $` produces an empty string, you may still want it to be skipped. That's why I suggested the `$host` idea.